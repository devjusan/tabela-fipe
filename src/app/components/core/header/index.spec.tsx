import { render, cleanup, screen } from '@testing-library/react';
import Header from '.';

describe('Header', () => {
  afterAll(() => {
    cleanup();
  });

  it('should match snapshot', () => {
    const headerText = 'Dark mode';
    render(<Header />);

    expect(screen.getByText(headerText)).toMatchSnapshot();
  });
});
