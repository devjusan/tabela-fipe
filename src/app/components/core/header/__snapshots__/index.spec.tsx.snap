// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<span
  class="MuiTypography-root MuiTypography-caption css-1sn4lm3-MuiTypography-root"
>
  Dark mode
</span>
`;
